tools/nite/main.cpp

```diff
@@ -0,0 +1,513 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#define _NBL_STATIC_LIB_
+#include <nabla.h>
+#include "nbl/video/utilities/CSimpleResizeSurface.h"
+
+#include "SimpleWindowedApplication.hpp"
+#include "CEventCallback.hpp"
+
+#include "nbl/ext/ImGui/ImGui.h"
+#include "nbl/ui/ICursorControl.h"
+
+// Test Engine
+#include "imgui_test_suite_imconfig.h"
+#include "imgui_test_suite.h"
+#include "imgui_te_engine.h"
+#include "imgui_te_ui.h"
+#include "imgui_te_utils.h"
+
+// Argparse
+#include <argparse/argparse.hpp>
+
+using namespace nbl;
+using namespace core;
+using namespace hlsl;
+using namespace system;
+using namespace asset;
+using namespace ui;
+using namespace video;
+
+class NITETool final : public examples::SimpleWindowedApplication
+{
+	using device_base_t = examples::SimpleWindowedApplication;
+	using clock_t = std::chrono::steady_clock;
+
+	_NBL_STATIC_INLINE_CONSTEXPR uint32_t WIN_W = 1280, WIN_H = 720, SC_IMG_COUNT = 3u, FRAMES_IN_FLIGHT = 5u;
```

Comment:
in C++ don't use `_NBL_STATIC_INLINE_CONSTEXPR` anymore, just write `constexpr static inline`